src/libs/OptionsListUtils.js

```diff
@@ -508,6 +509,18 @@ function getOptions(reports, personalDetails, activeReportID, {
         recentReportOptions = reportsSplitByDefaultChatRoom[0].concat(reportsSplitByDefaultChatRoom[1]);
     }
 
+    // If we are prioritizing 1:1 chats in search, do it only once we started searching
+    if (sortByReportTypeInSearch && searchValue !== '') {
+        recentReportOptions = lodashOrderBy(recentReportOptions, [(option) => {
+            if (option.isDefaultChatRoom || option.isArchivedRoom) {
+                return 3;
+            } if (!option.login) {
```

Comment:
Can you split this `if` to another line? I know it doesn't make a difference code-wise but it's a bit more consistent style-wise.

Reply:
Oops, it is supposed to be else if.

Reply:
No worries, eslint might say its a useless `else` b/c of return, if so feel free to put it on a new line

Reply:
just did.